core/trace/src/main/scala/org/typelevel/otel4s/trace/Tracer.scala

```diff
@@ -179,11 +180,9 @@ object Tracer {
   def apply[F[_]](implicit ev: Tracer[F]): Tracer[F] = ev
 
   trait Meta[F[_]] extends InstrumentMeta[F] {
-    def noopSpanBuilder: SpanBuilder.Aux[F, Span[F]]
-    final def noopResSpan[A](
-        resource: Resource[F, A]
-    ): SpanBuilder.Aux[F, Span.Res[F, A]] =
-      noopSpanBuilder.wrapResource(resource)
+    def noopSpanBuilder: SpanBuilder[F]
+    final def noopSpanRes: Resource[F, (Span[F], F ~> F)] =
+      noopSpanBuilder.build.resource
```

Comment:
is this method still useful at all?

Reply:
(no one replied so I'll assume that everyone agreed with me that the answer is "no")